### PR TITLE
feat: dip20 implementation

### PR DESCRIPTION
**Pull request by aleacevedo**
# feat: dip20 implementation

## Summary

- Make a big refactor to add standars easier
- Implement dip20 standard

### CODE REVIEW

File: src/utils/dfx/token/extMethods.ts [hunk 1]
```diff
@@ -26,15 +28,16 @@ const send = async (
 
   const transferResult = await actor.transfer(data);
 
-  if ('ok' in transferResult) return transferResult.ok;
+  if ('ok' in transferResult) return { amount: transferResult.ok };
```
Comment (rocky-fleek) [inline src/utils/dfx/token/extMethods.ts:31]: is this ok? does `ok` contain the transferred amount ?

**Comment by aleacevedo** [comment]
It was like this on merhods.ts before

File: src/utils/dfx/token/extMethods.ts (hunk 1 above)
Comment (aleacevedo) [reply src/utils/dfx/token/extMethods.ts:31]: It was that way on methods.ts

File: src/PlugWallet/index.ts [hunk 2]
```diff
@@ -90,7 +90,7 @@ class PlugWallet {
 
   principal: string;
 
-  registeredTokens: Array<StandardToken>;
+  registeredTokens: { [key: string]: StandardToken };
```
[the remark this answers is not in the document]
Comment (aleacevedo) [reply src/PlugWallet/index.ts:95]: Yep, I fix everything

**Pull request merged**